_includes/docs/latest/resource-coverage.md

```diff
@@ -2,16 +2,22 @@
 | Resource                                |
 | --------------------------------------- |
 | [App Engine Apps](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps) |
+| [App Engine Services](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services) |
+| [App Engine Version](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions) 
 | [BigQuery](https://cloud.google.com/bigquery/docs/reference/rest/v2/) |
 | [Billing Accounts](https://cloud.google.com/billing/reference/rest/v1/billingAccounts) |
 | [Buckets](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource) |
 | [Bucket Access Controls](https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls#resource) |
 | [Cloud APIs Configuration](https://cloud.google.com/apis/) |
 | [Cloud Audit Logging Configuration](https://cloud.google.com/iap/docs/audit-log-howto) |
-| [Cloud Dataproc Clusters](https://cloud.google.com/dataproc/docs/reference/rest/v1/projects.regions.clusters) |
+| [Cloud Dataproc Clusters](https://cloud.google.com/dataproc/docs/guides/create-cluster) |
```

Comment:
We should try to point this to the resource representation (i.e. the json).

Reply:
Fixed.